scripts/load-docker-compose.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# This will use docker-compose to load the data
+# Using a stage built main container and the actula 
+# Postgres container
+
+# Usage example:
+#	./load-docker-compose.sh -g kanindata-gotland-v9.xlsx \
+#	          -G herd-registry-gotland.xlsx \
+#	          -m kanindata-mellerud-v4.xlsx
+
+usage () {
+	cat <<-USAGE_END
+	Usage:
+	    $0	\\
+	    	-g kanindata-gotland.xlsx -G herd-registry-gotland.xlsx \\
+	    	-m kanindata-mellerud.xlsx
+
+	Options:
+
+	    -g file	Load Gotland rabbit data from "file"
+	    -G file	Load Gotland herd data from "file"
+	    -m file	Load Mellerud rabbit data from "file"
+
+	    -h		Show this help text
+	USAGE_END
+}
+
+
+while getopts 'g:G:hm:' opt; do
+	case $opt in
+		g)	gfile=$OPTARG ;;
+		G)	Gfile=$OPTARG ;;
```

Comment:
Naming? Could this be grabbit and gherd or grdata/ghdata or something?

Reply:
I just took @kusalananda script so need to update that instead